src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/ManageExternalMethodologyForSpecificPublicationAuthorizationHandlers.cs

```diff
@@ -0,0 +1,51 @@
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Security;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Microsoft.AspNetCore.Authorization;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers.AuthorizationHandlerUtil;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Security.SecurityClaimTypes;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
+{
+    public class ManageExternalMethodologyForSpecificPublicationRequirement : IAuthorizationRequirement
+    {
+    }
+
+    public class ManageExternalMethodologyForSpecificPublicationAuthorizationHandler 
+        : AuthorizationHandler<ManageExternalMethodologyForSpecificPublicationRequirement, Publication>
+    {
+        private readonly IUserPublicationRoleRepository _userPublicationRoleRepository;
+
+        public ManageExternalMethodologyForSpecificPublicationAuthorizationHandler(
+            IUserPublicationRoleRepository userPublicationRoleRepository)
+        {
+            _userPublicationRoleRepository = userPublicationRoleRepository;
+        }
+
+        protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context,
+            ManageExternalMethodologyForSpecificPublicationRequirement requirement,
+            Publication publication)
+        {
+            // If a Publication is linked to a Methodology, this should be unlinked first.
+            if (publication.Methodologies?.Count > 0)
```

Comment:
I wasn't sure how Methodologies is getting included here. Is the `PublicationMethodologies` guaranteed to be loaded at this point? I think we'd at least expect it to be an empty list if so, rather than allowing for the possibility of it to be null?

Reply:
Good shout - have added in an explicit Load of this property to be doubly sure.